api.md

```diff
@@ -950,6 +951,48 @@ to the server's timezone setting.
 
 ## Utilities/Statistics [](utilities)
 
+## timescaledb_information.hypertable [](timescaledb_information.hypertable)
+
+Get information about hypertables.
+
+#### Available Columns
+
+|Name|Description|
+|---|---|
+| `schema_name` | Schema name of the hypertable. |
+| `table_name` | Table name of the hypertable. |
+| `table_owner` | Owner of the hypertable. |
+| `num_dimensions` | Number of dimensions. |
+| `num_chunks` | Number of chunks. |
+| `table_bytes` |Disk space used by hypertable |
+| `index_bytes` |Disk space used by indexes|
+| `toast_bytes` |Disk space of toast tables|
+| `total_bytes` |Total disk space used by the specified table, including all indexes and TOAST data|
+
+#### Sample Usage
+
+Get information about all hypertables.
+
+SELECT * FROM timescaledb_information.hypertable;
+
+```sql
+SELECT * FROM timescaledb_information.hypertable;
+ table_schema | table_name | table_owner | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size 
+--------------+------------+-------------+----------------+------------+------------+------------+------------+------------
+ public       | metrics    | postgres    |              1 |          5 | 99 MB      | 96 MB      |            | 195 MB
```

Comment:
Just to clarify what a user would get with multiple hypertables in the DB, can you put one or two more example rows here?